packages/eth-providers/src/consts.ts

```diff
@@ -89,6 +89,7 @@ export const ORPHAN_TX_DEFAULT_INFO = {
 
 export const BLOCK_GAS_LIMIT = 29_990_016;
 export const BLOCK_STORAGE_LIMIT = 3_670_016;
+export const MAX_GAS_LIMIT_CC = 22;  // log2(BLOCK_STORAGE_LIMIT)
```

Comment:
```suggestion
export const MAX_GAS_LIMIT_CC = 21;  // log2(BLOCK_STORAGE_LIMIT)
```

2^22 > BLOCK_STORAGE_LIMIT